tests/test-gradle

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
+. "$SCRIPT_DIR/../poshlib/poshlib.sh" || exit 1
+use swine
+use parse-opt
+
+PO_SIMPLE_PARAMS=(DEMOS_BRANCH)
+eval $(parse-opt-simple)
+
+: "${DEMOS_BRANCH:=master}"
+
+tmpdir=$(mktemp -d)
+tmpfile=$(mktemp)
+
+cd "$tmpdir"
+git clone --recurse-submodules git@github.com:sirensolutions/siren-platform
+cd siren-platform
+
+gradleOpts=(-p benchmark -is \
+    -Pfederate.commit=8579c77702b13bc826e651db940b88d95bea76e5 \
```

Comment:
Maybe this commit could be a parameter with a default value as this depends on the ES version specified above (right now 7.9.2).

Reply:
done